### DISCUSSION TITLE
Why Received status code 400 from server ?

### DISCUSSION THREAD

**Discussion by Zuowxing** [trustwallet/wallet-core#2389] [Q&A]
I just want to run 'app'，But Gradle build error : Could not resolve com.trustwallet:wallet-core:2.9.5。...Username must not be null！So , I set the 'gpr.user'(my GitHub count) and 'gpr.key'(my GitHub token) in gradle.properties（Project Properties）-> rebuild -> run ->logical build error ,see the pic:
<img width="1294" alt="image" src="https://user-images.githubusercontent.com/105256563/179723510-01c692db-c00b-4a25-b1aa-5afcaba25913.png">

**Comment by hewigovens** [comment]
1. Check your internet, you might need VPN or proxy
2. Check if your GitHub token has package reading permission
